packages/web-react/src/components/UNSTABLE_Slider/__tests__/UNSTABLE_Slider.test.tsx

```diff
@@ -0,0 +1,52 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import UNSTABLE_Slider from '../UNSTABLE_Slider';
+
+describe('UNSTABLE_Slider', () => {
+  const id = 'slider-test';
+  const label = 'Slider';
+  const onChange = () => {};
+  const defaultValue = 10;
+
+  stylePropsTest((props) => <UNSTABLE_Slider id={id} {...props} data-testid="slider-test" />, 'slider-test');
+
+  restPropsTest((props) => <UNSTABLE_Slider id={id} {...props} />, 'div');
+
+  it('should render slider', () => {
+    render(<UNSTABLE_Slider id={id} value={defaultValue} onChange={onChange} label={label} />);
```

Comment:
You could have it like this:

```jsx
const defaultProps = {
  id: 'slider-test',
  label: 'Slider',
  onChange: () => {},
  value: defaultValue,
}

<UNSTABLE_Slider {...defaultProps} />
```

And later on in other tests:

```jsx
<UNSTABLE_Slider {...defaultProps} helperText='Helper text' />
```

Reply:
good idea!